### DISCUSSION TITLE
GUI question

### DISCUSSION THREAD

**Discussion by edugatei56** [dbeaver/dbeaver#10746] [General]
Hi there,

Have been using dbeaver for a year and two for my daily work, and I'm fairly happy with it, but I have sense there is something basic about the GUI that I'm missing. I feel dbeaver quickly leaves me with a boatload of tabs from various unrelated databases. What's worse: it can lead to confusing similarly named tables on different databases.

I typically have connections to my local server, a test server, and a live server. Each of these servers hold variuos databases for different projects. What I would expect is a way to easily show and hide all open tabs from a single database.
 https://trackeasy.fun/usps/
I work in SQL.

I hope someone can enligthen me - thanks!

**Comment by p91paul** [comment]
A possible workaround for this is described in https://github.com/dbeaver/dbeaver/discussions/10746
However, I think what you propose can make for a nice feature request!
